3p/3p.js

```diff
@@ -272,6 +272,7 @@ function validateAllowedFields(data, allowedFields) {
     location: true,
     mode: true,
     consentNotificationId: true,
+    consentHandlingOverride: true,
```

Comment:
this should be `blockOnConsent`, the attribute is `data-block-on-consent`

Reply:
done

Reply:
done

Reply:
I couldn't see the name change : )

Reply:
woops. really done now. thanks!